pkg/agent/syssetup/setup.go

```diff
@@ -12,18 +12,18 @@ import (
 
 func loadKernelModule(moduleName string) {
 	if _, err := os.Stat("/sys/module/" + moduleName); err == nil {
-		logrus.Infof("module %s was already loaded", moduleName)
+		logrus.Infof("Module " + moduleName + " was already loaded")
```

Comment:
Either switch to Info() or keep the formatting. I believe convention in this codebase is to use format not concatenation?